src/histograms/HistogramParams.ts

```diff
@@ -70,6 +70,10 @@ export class HistogramParams {
   public ticksDateFormat: string = null;
   public xAxisPosition: Position = Position.bottom;
   public descriptionPosition: Position = Position.bottom;
+  // Add 2 pixel around each label to detect overlap.
+  public overlapXTolerance  = 2;
+  // 4 is an arbitrary value to displayed one or 2 ticks between each value.
+  public tickNumbersOnResize  = 4;
```

Comment:
tickNumbersOnResize is more a muliplicator of the number of x labels used to change the number of ticks

So I would suggest : xLabelsToTicksFactor 

@MAudelGisaia @QuCMGisaia